checador/spec/requests/records_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe "Records", type: :request do
```

Comment:
I recommend you to add some specs to ensure this route is working properly
https://relishapp.com/rspec/rspec-rails/docs/request-specs/request-spec